src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/LspEditorFeatureDetector.cs

```diff
@@ -90,7 +109,69 @@ public bool IsLspEditorEnabled()
             _jtf.AssertUIThread();
         }
 
-        return !_lazyUseLegacyEditorTask.GetValue(_disposeTokenSource.Token);
+        _activityLog.LogInfo("Checking if LSP Editor is available");
+        if (documentMoniker is null)
+        {
+            _activityLog.LogWarning($"LSP Editor not available because {nameof(documentMoniker)} is null");
+            return false;
+        }
```

Comment:
Can remove this whole check by making `documentMonkier` non-nullable

Reply:
Done, thanks!